keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -163,19 +163,20 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  No. It's expected to accelerate scheduling a pod upon preemption, and hence it won't impact
+  already running workloads.
 
 * **What specific metrics should inform a rollback?**
+  The existing metric `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}`,
```

Comment:
5% is within the margin of error for filter latency, I would rephrase to say: "noticeable and sustainable increase in `scheduler_framework_extension_point_duration_seconds{extension_point="Filter"}`" latency metric

Reply:
Done